### PR TITLE
Merge duality neutron (CORRECT BRANCH)

### PR DESCRIPTION
**Pull request by jcompagni10**
Merge duality modules (dex, epochs, gmp, ibcswap & incentives) into neutron.

### CODE REVIEW

File: x/dex/types/tx.pb.go [hunk 1]
```diff
@@ -0,0 +1,5005 @@
...
+	}
+	return false
+}
+
+type MsgDeposit struct {
+	Creator         string                                   `protobuf:"bytes,1,opt,name=creator,proto3" json:"creator,omitempty"`
+	Receiver        string                                   `protobuf:"bytes,2,opt,name=receiver,proto3" json:"receiver,omitempty"`
+	TokenA          string                                   `protobuf:"bytes,3,opt,name=tokenA,proto3" json:"tokenA,omitempty"`
+	TokenB          string                                   `protobuf:"bytes,4,opt,name=tokenB,proto3" json:"tokenB,omitempty"`
+	AmountsA        []github_com_cosmos_cosmos_sdk_types.Int `protobuf:"bytes,5,rep,name=amountsA,proto3,customtype=github.com/cosmos/cosmos-sdk/types.Int" json:"amountA" yaml:"amountsA"`
```
Comment (swelf19) [inline x/dex/types/tx.pb.go]: go struct name - Amount**s**A
proto field name - amount**s**A
and json field name - amountA (singular)
Isnt it a good time to make names consistent

**Comment by pr0n00gler** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6734653064

File: x/dex/types/tx.pb.go (hunk 1 above)
Comment (jcompagni10) [reply x/dex/types/tx.pb.go]: 👍

**Comment by pr0n00gler** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6802727604

**Comment by pr0n00gler** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6865972417

**Comment by pr0n00gler** [comment]
https://github.com/neutron-org/neutron-tests/actions/runs/6990462605

**Pull request merged**